bin/git/getCommitAndBranch.js

```diff
@@ -17,8 +17,14 @@ export async function getCommitAndBranch({ inputFromCI } = {}) {
     TRAVIS_REPO_SLUG,
     TRAVIS_PULL_REQUEST_SHA,
     TRAVIS_PULL_REQUEST_BRANCH,
+    GITHUB_WORKFLOW,
```

Comment:
Is this an environment variable we ask users to set?

Reply:
No, github sets it.

Reply:
Always?

Reply:
in a github action, yes